code/modules/power/cable.dm

```diff
@@ -29,14 +29,14 @@ GLOBAL_LIST_INIT(wire_node_generating_types, typecacheof(list(/obj/structure/gri
 /obj/structure/cable/layer1
 	color = "red"
 	cable_layer = CABLE_LAYER_1
-	machinery_layer = null
-	layer = WIRE_LAYER - 0.01
-	icon_state = "l1-1-2-4-8-node"
+	machinery_layer = MACHINERY_LAYER_1 // <- original null, makes different colored wired not usesless and can connect to machines
+	layer = WIRE_LAYER - 0.01			// possibly can make it so machines can have it's layered adjusted in-game
+	icon_state = "l1-1-2-4-8-node"		// furthermore, the multilayer hub (non z) does not actually connect machines on top of it
```

Comment:
Could you elaborate on the things you're saying about cable hubs? was anything related to them changed here?

Reply:
> It might be better to split this into 2 prs if they aren't related.

You're right it is two separate updates. I'll make these into two different ones.

> On another note, please run your map changes through mapmerge https://github.com/tgstation/tgstation/wiki/Map-Merger-2%3A-The-Better-One, that's what the linter is complaining about.

I'll take a look and do this on a new PR.

Reply:
> Could you elaborate on the things you're saying about cable hubs? was anything related to them changed here?

Nothing was changed about the hubs, I was just noting that it looks like the Hub was being constructed so that it would be able to sit under a machine, like an SMES, and distribute power through multiple layers. This would be great, however the hub doesn't connect to the machine in its tile.

There are also `machinery_layer` vars that would suggest that there could be a feature where you can adjust the machine to connect to a different layer. I set both red and blue cables to the same `machinery_layer_1` as the regular cable for now, allowing multi-layered cables to be useable.

Reply:
Use cable hub in places where you need connect cable to machine.
I make  non central cables to not connect to machinery to be able  place it under machinery.  
![(null)scrnshot1](https://user-images.githubusercontent.com/7734424/99198242-12e23f80-27a0-11eb-8995-9f6010dc772c.png)

Reply:
> Use cable hub in places where you need connect cable to machine.
> I make non central cables to not connect to machinery to be able place it under machinery.

The point of this change is to allow machinery and terminals connect to any layer of cable, cleaning up the mess of having a ton of hubs. The hubs are good for distributing power to the other layers, however currently you need to have another hub at every terminal to layer it up. This creates an ugly setup of wires and overcomplicates wiring. I do think that the end wires could use some spriting when connecting to terminals so that the end is hidden under it. Also placing a hub under an SMES does not provide power to the cables. This change does not affect that fact, but does allow those layered cables to get power from the SMES.

![smwa](https://user-images.githubusercontent.com/1571911/99206315-f0d0c900-276f-11eb-87f4-1f12b0aa1cac.png)

Reply:
> placing a hub under an SMES does not provide power to the cables

In setup that i show upper it provide power. 
You place hub by mapping?